plugins/woocommerce-admin/client/marketplace/contexts/types.ts

```diff
@@ -17,6 +23,10 @@ export type MarketplaceContextType = {
 	addInstalledProduct: ( slug: string ) => void;
 	hasBusinessServices: boolean;
 	setHasBusinessServices: ( hasBusinessServices: boolean ) => void;
+	searchResultsCount: SearchResultsCountType;
+	setSearchResultsCount: (
+		cb: ( prevState: SearchResultsCountType ) => SearchResultsCountType
```

Comment:
(no need to change) - personally I'm not a fan of those short names and would just use `callback` or `setState` here

Reply:
Fixed in 6c2c4bfa07 :)